ckanext/depositar_theme/fanstatic/scripts/alert.js

```diff
@@ -0,0 +1,4 @@
+(function() {
+  let alertEl = document.getElementsByClassName("alert-dismissible");
```

Comment:
優先使用單引號 (其他 scripts 資料夾下的 js 檔也請一併修改)

```suggestion
  let alertEl = document.getElementsByClassName('alert-dismissible');
```